Revised metadata for the SDN managed by on-premises tools docs

ManikaDhiman opened the pull request.

v-regandowner commented:
@alkohli - Can you review the proposed changes? 
	
IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

ManikaDhiman commented on azure-local/concepts/plan-software-defined-networking-infrastructure-23h2.md:
```diff
@@ -1,6 +1,6 @@
 ---
-title: Plan a Software Defined Network infrastructure for Azure Local, version 23H2
-description: This topic provides information on how to plan a Software Defined Network (SDN) infrastructure deployment for Azure Local, version 23H2.
+title: Plan infrastructure for Software Defined Network managed by on-premises tools in Azure Local, version 23H2
```
Changed to Software Defined Networking.

ManikaDhiman commented on azure-local/concepts/route-reflector-overview.md:
```diff
@@ -1,6 +1,6 @@
-title: Overview of BGP Route Reflector in Azure Localand Windows Server
-description: Use this topic to learn about BGP Route Reflector for Software Defined Networking in Azure Local and Windows Server.
+title: Overview of BGP Route Reflector in Azure Local and Windows Server
+description: Use this topic to learn about BGP Route Reflector for Software Defined Networking managed by on-premises tools in Azure Local and Windows Server.
```
No change. It is Software Defined Networking.

ManikaDhiman commented on azure-local/deploy/sdn-wizard-23h2.md:
```diff
@@ -1,18 +1,18 @@
-title: Deploy SDN using Windows Admin Center for Azure Local
-description: Learn how to deploy an SDN infrastructure using Windows Admin Center for Azure Local
+title: Deploy Software Defined Network managed by on-premises tools using Windows Admin Center for Azure Local
+description: Learn how to deploy infrastructure for Software Defined Network managed by on-premises tools using Windows Admin Center for Azure Local
```
Updated to Software Defined Networking.

ManikaDhiman commented on azure-local/deploy/sdn-wizard-23h2.md:
```diff
@@ -1,18 +1,18 @@
 [!INCLUDE [applies-to](../includes/hci-applies-to-23h2.md)]
 
-This article describes how to deploy Software Defined Networking (SDN) through Windows Admin Center after you deployed your Azure Local via the Azure portal.
+This article describes how to deploy Software Defined Networking (SDN) managed by on-premises tools through Windows Admin Center after you deployed your Azure Local via the Azure portal.
```
Updated.

ManikaDhiman commented on azure-local/manage/assign-public-ip-to-vm.md:
```diff
@@ -1,9 +1,9 @@
 ---
 title: Assign a public IP address to a virtual machine
-description: Learn how to assign a public IP address to a virtual machine in an SDN environment.
```
Expended SDN in description.

ManikaDhiman commented:
> @ManikaDhiman - Great work as always! Given the time frame you may want to make the ms.date something like today lol.
> 
> The only thing you'll see is whether to reference throughout articles as Software Defined Networking or Software Defined Network in the titles and descriptions mostly.

Hi @ronmiab: Thank you so much for the thorough review. I've incorporated your feedback and fixed merge conflicts. Thanks again!
